chpl/chpl-api/src/main/resources/log4j2-xinclude-loggers.xml

```diff
@@ -0,0 +1,150 @@
+        additivity="false">
+        <AppenderRef ref="apiKeyWarningEmailJob" />
+    </Logger>
+    <Logger name="apiKeyDeleteJobLogger" level="INFO"
+        additivity="false">
+        <AppenderRef ref="apiKeyDeleteJob" />
+    </Logger>
+    <Logger name="questionableActivityEmailJobLogger" level="INFO"
+        additivity="false">
+        <AppenderRef ref="questionableActivityEmailJob" />
+    </Logger>
+    <Logger name="pendingChangeRequestEmailJobLogger" level="INFO"
+        additivity="false">
+        <AppenderRef ref="pendingChangeRequestEmailJob" />
+    </Logger>
+    <Logger name="interruptJobLogger" level="INFO"
+        additivity="false">
+        <AppenderRef ref="interruptJob" />
+    </Logger>
+    <Logger name="massRequirePasswordChangeJobLogger" level="INFO"
+        additivity="false">
+        <AppenderRef ref="massRequirePasswordChangeJob" />
+    </Logger>
+    <Logger name="updateListingStatusJobLogger" level="INFO"
+        additivity="false">
+        <AppenderRef ref="updateListingStatusJob" />
+    </Logger>
+    <Logger name="realWorldTestingEligibilityJobLogger" level="INFO"
+        additivity="false">
+        <AppenderRef ref="realWorldTestingEligibilityJob" />
+    </Logger>
+    <Logger name="developerAccessEmailJobLogger" level="INFO"
+        additivity="false">
+        <AppenderRef ref="developerAccessEmailJob" />
+    </Logger>
+    <Logger name="directReviewDownloadableResourceCreatorJobLogger" level="INFO"
+        additivity="false">
+        <AppenderRef ref="directReviewDownloadableResourceCreatorJob" />
+    </Logger>
+    <Logger name="directReviewCacheRefreshJobLogger" level="INFO"
+        additivity="false">
+        <AppenderRef ref="directReviewCacheRefreshJob" />
+    </Logger>
+    <Logger name="realWorldTestingUploadJobLogger" level="INFO"
+        additivity="false">
+        <AppenderRef ref="realWorldTestingUploadJob" />
+    </Logger>
+    <Logger name="realWorldTestingReportEmailJobLogger" level="INFO"
+        additivity="false">
+        <AppenderRef ref="realWorldTestingReportEmailJob" />
+    </Logger>
+    <Logger name="auditDataRetentionJobLogger" level="INFO"
+        additivity="false">
+        <AppenderRef ref="auditDataRetentionJob" />
+    </Logger>
+    <Logger name="listingValidationReportCreatorJobLogger" level="INFO"
+        additivity="false">
+        <AppenderRef ref="listingValidationReportCreatorJob" />
+    </Logger>
+    <Logger name="listingValidationReportEmailJobLogger" level="INFO"
+        additivity="false">
+        <AppenderRef ref="listingValidationReportEmailJob" />
+    </Logger>
+    <Logger name="quarterlyReportGenerationJobLogger" level="INFO"
+        additivity="false">
+        <AppenderRef ref="quarterlyReportGenerationJob" />
+    </Logger>
+    <Logger name="annualReportGenerationJobLogger" level="INFO"
+        additivity="false">
+        <AppenderRef ref="annualReportGenerationJob" />
+    </Logger>
+</Loggers>
```

Comment:
The auto-checker called this one out. Not blocking approval, but it ought to be updated at some point